index.html

```diff
@@ -12,6 +13,9 @@
     <meta name="image" content="http://cornelius.patatap.com/images/thumbnail.jpg">
     <meta name="description" content="Patatap is a portable animation and sound kit. With the touch of a finger create melodies charged with moving shapes. Warning: contains flashing images.">
 
+    <meta name="mobile-web-app-capable" content="yes" />
```

Comment:
I think these are not necessary if you have the manifest file but I put them just in case...